src/editors/containers/EditorContainer/components/EditorFooter/index.jsx

```diff
@@ -28,7 +28,7 @@ export const EditorFooter = ({
       </Toast>
     )}
 
-    <ModalDialog.Footer>
+    <ModalDialog.Footer className="border-top-0">
```

Comment:
Visual fix for this line above cancel and save footer buttons.

Reply:
Is the border line not supposed to be there in the hi-fi mocks? Just curious